misc/packaging/README.md

```diff
@@ -0,0 +1,117 @@
+# ubuntuSoS
+this scripts show how to build acrn-hypervisor and acrn-kernel deb package
+
+1. setup native ubuntu
+
+install python3
+
+config proxy
+
+sudo su
```

Comment:
why is this command needed?

Reply:
if do not have superuser right will interrupt the whole install process, e.g input password.... so need this right